broker-core/src/main/java/io/zeebe/broker/subscription/message/state/MessageSubscriptionState.java

```diff
@@ -149,21 +146,18 @@ public void resetSentTime(MessageSubscription subscription) {
   }
 
   public void updateSentTime(final MessageSubscription subscription, long sentTime) {
-    dbContext.runInTransaction(
```

Comment:
This and visitSubscriptions are called by the PendingMessageSubscriptionChecker which doesn't seem to be running in a transaction, although I might be understanding it incorrectly.

Reply:
True for update sentTime I still have the transaction and for the visit internally we always ensure that is run in a transaction.